### DISCUSSION TITLE
[Idea] Re-authorise the editing of combinations

### DISCUSSION THREAD

**Discussion by Touxten** [PrestaShop/PrestaShop#36686] [Ideas]
### The solution

Since 1.7 we can no longer edit a combination, we have to delete it and create it again, which poses problems in the event of a return because the stock will no longer be good.

Acutally I use a personal script, but it's a shame that this functionality is no longer available. 

![Screenshot (26)](https://github.com/user-attachments/assets/5ca6b298-41fc-4c87-9fe7-6ac405436470)

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by Hlavtox** [comment]
What is a declination?

**Reply by matks** [reply]
I guess a combination but combinations can be edited

**Comment by matks** [comment]
@Touxten Can you explain more what you mean?

In the latest version of PrestaShop, PrestaShop 8.1.0
- attributes can be edited (like "Color" or "Size")
- attribute values (like "Red", "Blue", "Green") can be edited
- combinations, generated based on attributes (like color, size...) can be edited

**Reply by Touxten** [reply]
Combinations, generated based on attributes (like color, size...) can be edited but you can't change color or size.

**Reply by Touxten** [reply]
[screen-capture.webm](https://github.com/user-attachments/assets/c4e8e8e2-181e-4e64-bc02-6ea1140df954)

**Comment by Touxten** [comment]
Hey @matks can you confirm it's possible i don't see where ?

**Reply by matks** [reply]
So when you say "edit" you mean "modify the attribute on which the combination is based" 😉

**Reply by Touxten** [reply]
Yes! You can't edit a combination like in my video!
It's a big step backwards

It's not a problem for me, I continue to use my own script.

I don't like your answer you understood very well. 

Good day I close the discussion
